docs/CNAME

```diff
@@ -0,0 +1 @@
+neurobagel.org
```

Comment:
I read this https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain as saying "add your www subdomain" - what do you think?